docs/advanced/cloudinitcrd.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 10
+sidebar_label: CloudInit CRD
+title: "CloudInit CRD"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/cloudinitcrd"/>
+</head>
+
+_Available as of v1.3.0_
+
+Harvester now offers the capability to configure Harvester OS specific settings via the `CloudInit` CRD.
+
+This can be used to configure underlying Harvester OS specific options manually or via Gitops
+
+## Background
+Harvester OS uses the [elemental-toolkit](https://rancher.github.io/elemental-toolkit/docs/reference/cloud_init/) which ships with its unique cloud-init support. 
+
+As part of the Harvester installation, settings defined by users are written to an elemental cloud-init file in /oem directory. As Harvester OS itself is immutable, the cloud-init file ensures node specific settings are applied on each reboot. 
+
+The Harvester `CloudInit` CRD exposes the `elemental cloud-init` via a Kubernetes CRD. This allows users to perform additional node specific changes post Harvester install without having to follow the process of making the rootfs writable.
+
+The `CloudInit` CRD is persisted and sync across to the underlying hosts via Harvester, which ensures changes made directly to OS are not lost across reboots or upgrades.
+
+:::note
+
+`CloudInit CRD` is a cluster scoped resource. Please ensure you have appropriate access via Rancher rbac to access the resource.
+
+:::
```

Comment:
```suggestion
## Background

The Harvester operating system uses the [elemental-toolkit](https://github.com/rancher/elemental-toolkit), which has a unique form of [cloud-init support](https://rancher.github.io/elemental-toolkit/docs/reference/cloud_init/). 

Settings configured during the Harvester installation process are written to the `elemental` cloud-init file in the `/oem` directory. Because the Harvester operating system is immutable, the cloud-init file ensures that node-specific settings are applied on each reboot. 

The Harvester `CloudInit` CRD exposes the cloud-init file through a Kubernetes CRD. This allows you to modify node-specific settings after installation without needing to take steps to make the root filesystem writable.

In addition, the `CloudInit` CRD is persisted and synchronized with the underlying hosts so that changes made directly to the Harvester operating system are not lost whenever nodes are rebooted and upgraded.

:::note

The `CloudInit` CRD is a cluster-scoped resource. Ensure that your user account has the permissions required to access the resource (via Rancher role-based access control).

:::
```